### DISCUSSION TITLE
Combobox component strange behaviour

### DISCUSSION THREAD

**Discussion by LechK** [twilio-labs/paste#2716] [General] [answered]
Hello. Noticed strange behavior. If we want to use reset, we have to use useComboBox hook, in this hook we have to pass our items.
AND in ComboBox we HAVE to pass items again. so if we do that, after we pass array to combobox in two places, on mouse move after scroll, we automatically are scrolled to top of our list.
Can you check that?

`const { reset, ...props } = useCombobox({
    items: filtered,
  });

  return (
    <Combobox
      state={{ reset, props }}
      autocomplete
      items={filtered}
      labelText="Select Time Zone"
      placeholder="America/chicago"
      insertAfter={
          <Box>
            <ProductRegionalIcon decorative />
          </Box>
        )
      }`
      
I mean, if we passing into state by useComboBox, why we still have to pass into Combobox again?

**Comment by ghost** [comment, marked as answer]
Thanks for the report here @LechK could you put together a reproduction example using [our codesandbox template](https://codesandbox.io/s/paste-bug-report-j8jxpf)? that would help us diagnose the issue you're seeing with greater effectiveness! thank you

**Reply by LechK** [reply]
Strange, I can't reproduce that in sandbox, it works without any problems, maybe in my code it conflicts somehow with other things. Thank You! Probably we can close that!

**Reply by nkrantz** [reply]
Feel free to update this thread if you have any problems with this in the future!
